feature (no issue) Contributor's Guide update Add Use Case instructions

georgemccabe opened the pull request:
## Pull Request Testing ##

- [X] Describe testing already performed for these changes:</br>
Rendered documentation and reviewed output for formatting issues:
http://icculus.rap.ucar.edu/metplus-contrib/Contributors_Guide/add_use_case.html

- [X] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>
Review changes and rendered content to verify that everything looks correct.

- [X] Do these changes include sufficient documentation and testing updates? **[Yes]**
It is only documentation updates.

- [X] Will this PR result in changes to the test suite? **[No]**</br>
If **yes**, describe the new output and/or changes to the existing output:</br>

## Pull Request Checklist ##
See the [METplus Workflow](https://dtcenter.github.io/METplus/Contributors_Guide/github_workflow.html) for details.
- [X] Complete the PR definition above.
- [X] Ensure the PR title matches the feature or bugfix branch name.
- [X] Define the PR metadata, as permissions allow.
Select: **Reviewer(s)**, **Project(s)**, and **Milestone**
- [ ] After submitting the PR, select **Linked Issues** with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue and delete your feature or bugfix branch from GitHub.

georgemccabe committed:
updated content for adding new use case to be more clear

georgemccabe committed:
renamed use case category ID to be more descriptive

georgemccabe committed:
moved images to correct location

georgemccabe committed:
Merge branch 'develop' into feature_contrib_guide_add_use_case

georgemccabe committed:
clean up content and add instruction to verify content of tarfile is correct

georgemccabe committed:
added note to be careful if there are multiple feature branches that include data from the same category to avoid losing data

georgemccabe committed:
Merge branch 'feature_contrib_guide_add_use_case' of github.com:dtcenter/METplus into feature_contrib_guide_add_use_case

jprestop commented on docs/Contributors_Guide/add_use_case.rst:
```diff
@@ -559,18 +706,25 @@ pull request is merged so that the develop branch will contain the new data.
-    ln -s ${DATA_TARFILE_DIR}/${METPLUS_VERSION}/sample_data-${USE_CASE_CATEGORY}-${METPLUS_VERSION}.tgz sample_data-${USE_CASE_CATEGORY}.tgz
+    ln -s ${METPLUS_DATA_TARFILE_DIR}/${METPLUS_VERSION}/sample_data-${METPLUS_USE_CASE_CATEGORY}-${METPLUS_VERSION}.tgz sample_data-${METPLUS_USE_CASE_CATEGORY}.tgz
 
 - Merge the pull request and verify that all of the Travis-CI tests pass for
```
```suggestion
- Merge the pull request and verify that all of the Travis-CI tests pass for
```
Should this be changed to GitHub Actions tests?

jprestop replied:
```suggestion
- Merge the pull request and verify that all of the GitHub Actions tests pass for
```

georgemccabe committed:
Update docs/Contributors_Guide/add_use_case.rst

Co-authored-by: jprestop <jpresto@ucar.edu>

jprestop reviewed (commented):
There are about 44 references to "you" on this page.  Not all are introduced here in this update, but some are.  We have been trying to keep the documentation in third person.  For example, instead of saying "Open this file with your favorite editor and modify it to include the appropriate information for your use case." we might say "Open this file with any editor and modify it to include appropriate information for the new use case."  Would you be able to fix up this file to make it from a 3rd person perspective?  If not, I understand, and can go ahead and approve this pull request.

georgemccabe commented:
> 
> 
> There are about 44 references to "you" on this page. Not all are introduced here in this update, but some are. We have been trying to keep the documentation in third person. For example, instead of saying "Open this file with your favorite editor and modify it to include the appropriate information for your use case." we might say "Open this file with any editor and modify it to include appropriate information for the new use case." Would you be able to fix up this file to make it from a 3rd person perspective? If not, I understand, and can go ahead and approve this pull request.

I realized that after I had already started writing content. Towards the end I switched to the correct perspective. I figured there is an open task to fix the perspective for all of the docs so I didn't think it would be a good use of my time to go back and clean the existing content up now. I have another big task I need to get wrapped up before the beta release and I switched over to this task because I realized more people are creating new use cases and updating the content now will make their work easier. Sorry!

jprestop commented:
I figured that would likely be the case, but also figured it wouldn't hurt to ask.  :)  Once, the checks are complete, I will approve this pull request.

jprestop reviewed (approved):
I have reviewed the changes and approve this pull request.  Once approved, it is ready to be merged by you, the requestor (only mentioned because we just changed this procedure).
